tests/safety/test_volkswagen.py

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env python3
```

Comment:
tests are good. played with them and any change to params caused a test failure, as expected.